deploy/charts/cert-manager/values/go.mod

```diff
@@ -0,0 +1,7 @@
+module main
+
+go 1.16
+
+replace github.com/alecthomas/jsonschema => github.com/amurant/jsonschema v0.0.0-20210729111349-9c856f912aa1
+
+require github.com/amurant/helm-jsonschema-gen v0.0.1
```

Comment:
Cool project @inteon I haven't gone and reviewed the code yet, but perhaps you should share this with the Helm maintainers.